protected/humhub/modules/content/models/ContentContainerDefaultPermission.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\content\models;
+
+/**
+ * This is the model class for table "contentcontainer_default_permission".
+ *
+ * @property string $permission_id
+ * @property string $contentcontainer_class
+ * @property string $group_id
+ * @property string $module_id
+ * @property string $class
+ * @property integer $state
+ */
+class ContentContainerDefaultPermission extends \yii\db\ActiveRecord
+{
+
+    /**
+     * @inheritdoc
+     */
+    public static function tableName()
+    {
+        return 'contentcontainer_default_permission';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function rules()
+    {
+        return [
+            [['permission_id', 'contentcontainer_class', 'group_id', 'module_id'], 'required'],
+            [['state'], 'integer'],
+            [['permission_id'], 'string', 'max' => 150],
+            [['contentcontainer_class', 'class'], 'string', 'max' => 255],
+            [['group_id', 'module_id'], 'string', 'max' => 50],
+        ];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function attributeLabels()
```

Comment:
It should be ok to just delete this method?

Reply:
yes, I don't find where it is used, there is no separate page with edit a single permission, so the labels are not used. I have removed the method and same for the class `ContentContainerPermission`.
Commit https://github.com/humhub/humhub/pull/4678/commits/ff112847b8b9cc735389b8da5759be619598fcd7.